integration/common/openlineage/common/provider/dbt.py

```diff
@@ -0,0 +1,202 @@
+import json
+import yaml
+import os
+import uuid
+from typing import List, Tuple, Dict, Optional
+
+import attr
+
+from openlineage.client.facet import DataSourceDatasetFacet, SchemaDatasetFacet, SchemaField, \
+    SqlJobFacet
+from openlineage.common import __version__
+from openlineage.client.run import RunEvent, RunState, Run, Job, Dataset
+from openlineage.common.utils import get_from_nullable_chain
+
+PRODUCER = f"openlineage-dbt/{__version__}"
```

Comment:
It looks like the convention for DBT adapters is `dbt-<adapter>` (see [Community Supported](https://docs.getdbt.com/docs/available-adapters#community-supported)). So we'll want to use `dbt-openlineage`

Reply:
Yes, but we're not dbt adapter here - I'm using our producer terminology, as `openlineage-airflow`. What do you think?

Reply:
Sounds good 👍